modules/registry-configuring-registry-storage-rhodf-cephrgw.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * registry/configuring_registry_storage/configuring-registry-storage-vsphere.adoc
+//
+// * registry/configuring_registry_storage/configuring-registry-storage-baremetal.adoc
+//
+// * registry/configuring_registry_storage/Configuring-the-registry-for-rhodf.adoc
+
+:_content-type: PROCEDURE
+[id="registry-configuring-registry-storage-rhodf-cephrgw_{context}"]
+= Configuring the Image Registry Operator to use Ceph RGW storage with Red Hat OpenShift Data Foundation
+
+{rh-storage-first} integrates multiple storage types that you can use with the internal image registry:
+
+* Ceph, a shared and distributed file system and on-premises object storage
+* NooBaa, providing a Multicloud Object Gateway
+
+This procedure outlines the procedure to  configure the image registry to use Ceph RGW storage.
+
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have access to the {product-title} web console.
+* You installed the `oc` CLI.
+* Install the {rh-storage-first} Operator to provide object storage and `cephfs` file storage.
+
+.Procedure
+
+. Create the object bucket claim using the `ocs-storagecluster-ceph-rgw` storage class. For example:
++
+[source,terminal]
+----
+cat <<EOF | oc apply -f -
+apiVersion: objectbucket.io/v1alpha1
+kind: ObjectBucketClaim
+metadata:
+  name: rgwtest
+  namespace: openshift-storage
+spec:
+  storageClassName: ocs-storagecluster-ceph-rgw
+  generateBucketName: rgwtest
+EOF
+----
+
+. Get the bucket name by entering the following command:
++
+[source,terminal]
+----
+$ bucket_name=$(oc get obc rgwtest -o jsonpath='{.spec.bucketName}')
```

Comment:
need add namespace in cmd
`bucket_name=$(oc get obc -n openshift-storage rgwtest  -o jsonpath='{.spec.bucketName}')`